# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/sharutils/nls-install.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- sharutils-4.2.1/po/Makefile.in.in~	1995-12-04 17:19:48.000000000 +0100
+++ sharutils-4.2.1/po/Makefile.in.in	2003-06-20 13:05:32.000000000 +0200
@@ -117,14 +117,14 @@
 install-exec:
 install-data: all
 	$(top_srcdir)/mkinstalldirs $(datadir)
-	catalogs='$(CATALOGS)'; \
+	catalogs='$(GMOFILES)'; \
 	for cat in $$catalogs; do \
 	  if test "`echo $$cat | sed 's/.*\(\..*\)/\1/'`" = ".gmo"; then \
 	    destdir=$(gnulocaledir); \
 	  else \
 	    destdir=$(localedir); \
 	  fi; \
-	  lang=`echo $$cat | sed 's/$(CATOBJEXT)$$//'`; \
+	  lang=`echo $$cat | sed 's/\.[^.]*$$//'`; \
 	  dir=$$destdir/$$lang/LC_MESSAGES; \
 	  $(top_srcdir)/mkinstalldirs $$dir; \
 	  if test -r $$cat; then \
